spaceship.zsh

```diff
@@ -210,6 +215,26 @@ spaceship_ruby_version() {
   echo -n "%{$reset_color%}"
 }
 
+# Show current vi_mode mode
+spaceship_vi_mode() {
```

Comment:
Can you please add this function (you've already mentioned): 

```sh
function zle-keymap-select() { zle reset-prompt; zle -R; };
zle -N zle-keymap-select;
bindkey -v
```

I think, that would be useful for debugging purposes. Name it like `spaceshipt_enable_vimode` or something.